packages/flutter/lib/src/widgets/widget_span.dart

```diff
@@ -139,7 +139,8 @@ class WidgetSpan extends PlaceholderSpan {
 
   @override
   int? codeUnitAtVisitor(int index, Accumulator offset) {
-    return null;
+    offset.increment(1);
+    return PlaceholderSpan.placeholderCodeUnit.codeUnitAt(0);
```

Comment:
Are there any implications for doing this?  Like I wonder, in the future if we support copying a WidgetSpan, will the plain text representation of the WidgetSpan be `\uFFFC`?

Reply:
This method is mainly used to calculate layout geometry. I would imagine the copying use the toPlainText instead, or something like toRichText. It has to return something when this method is called, returning null at index within a string seems counter-intuitive.

Reply:
Sounds good, it shouldn't matter at all for copying then.  The `\uFFFC` character makes sense to me here, I think that's what I was using as a placeholder when messing around with rich text editing.